_docs/DOC_SCAN_SANDBOX.md

```diff
@@ -0,0 +1,25 @@
+# Yoti Go Doc Scan Sandbox Module
+
+This module contains the tools you need to test your Go back-end integration with the Doc Scan Sandbox service.
+
+## Importing the Sandbox
+
+You can reference the sandbox by adding the following import:
+
+```Go
+import "github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox"
+```
+
+## Configuration
+The sandbox is initialised in the following way:
+```Go
+sandboxClient := sandbox.NewClient(sandboxClientSdkId, privateKey)
```

Comment:
I _think_ I made the profile Sandbox take `*rsa.PrivateKey` instead of []byte, because the bytes will be disposed once the key is read, and we'll need the key again when we initialise the regular Yoti Client. Will update this if that is the case, once I test it out

Reply:
Just looked at the sandbox profile example and we currently show:
```go
privateKey, err := cryptoutil.ParseRSAKey(pemFileBytes)
	assert.NilError(t, err)

	sandboxClient := sandbox.Client{
		ClientSdkID: sandboxClientSdkId,
		Key:         privateKey,
	}
```
For setting up the profile sandbox, and then the pemFileBytes are used to initialise the YotiClient, so all is OK.

Maybe we can introduce the NewClient to the profile Sandbox, and then clients have the option to specify the bytes as well.

Reply:
Created SDK-1677 for introducing NewClient to profile Sandbox too